### PR TITLE
Fix issue #6 and #10

### PR DESCRIPTION
**Pull request by Tiihott**
Attempt at fixing issues #6 and #10

### CODE REVIEW

**Review by kortemik — APPROVED** [review]
fixes issue #6 and #10

File: src/main/java/com/teragrep/jpr_01/JavaPcre.java [hunk 1]
```diff
@@ -169,10 +169,10 @@ public static class ByValue extends RegexStruct implements Structure.ByValue {}
     LibJavaPcre.OptionsStruct compile_options;
     LibJavaPcre.ExtraOptionsStruct extra_options;
     LibJavaPcre.MatchOptionsStruct match_options;
-    Map<String, Integer> name_table;
-    Map<Integer, String> match_table;
+    public Map<String, Integer> name_table;
```
Comment (kortemik) [inline src/main/java/com/teragrep/jpr_01/JavaPcre.java:172]: instead of direct member field access, change to getter pattern

**Review by kortemik — CHANGES_REQUESTED** [review]

File: src/main/java/com/teragrep/jpr_01/JavaPcre.java (hunk 1 above)
Comment (kortemik) [reply src/main/java/com/teragrep/jpr_01/JavaPcre.java:172]: please also make all member fields private, if not absolutely required to be package-private

**Comment by kortemik** [comment]
please move [JavaPcreTest.java](https://github.com/teragrep/jpr_01/blob/main/src/test/java/com/teragrep/JavaPcreTest.java) to jpr_01.tests

**Comment by Tiihott** [comment]
All fields should now be set to private with added get-functions. Also removed "pcre2_" from function names where it is not needed for better clarity.

**Review by kortemik — APPROVED** [review]
Requested changes seem to be implemented correctly. Thanks a lot!

**Pull request merged**